aws-bundle/LICENSE

```diff
@@ -297,220 +297,215 @@ License: CC0 - http://creativecommons.org/publicdomain/zero/1.0/
 Project URL: https://aws.amazon.com/sdkforjava
 License: Apache License, Version 2.0 - https://aws.amazon.com/apache2.0
 
 --------------------------------------------------------------------------------
 
-Group: software.amazon.awssdk  Name: regions  Version: 2.28.5
+Group: software.amazon.awssdk  Name: regions  Version: 2.30.6
 License: Apache License, Version 2.0 - https://aws.amazon.com/apache2.0
 
 --------------------------------------------------------------------------------
 
-Group: software.amazon.awssdk  Name: s3  Version: 2.28.5
+Group: software.amazon.awssdk  Name: s3  Version: 2.30.6
 Project URL: https://aws.amazon.com/sdkforjava
 License: Apache License, Version 2.0 - https://aws.amazon.com/apache2.0
 
 --------------------------------------------------------------------------------
 
-Group: software.amazon.awssdk  Name: sdk-core  Version: 2.28.5
+Group: software.amazon.awssdk  Name: sdk-core  Version: 2.30.6
 Project URL: https://aws.amazon.com/sdkforjava
 License: Apache License, Version 2.0 - https://aws.amazon.com/apache2.0
 
 --------------------------------------------------------------------------------
 
-Group: software.amazon.awssdk  Name: sso  Version: 2.28.5
+Group: software.amazon.awssdk  Name: sso  Version: 2.30.6
 Project URL: https://aws.amazon.com/sdkforjava
 License: Apache License, Version 2.0 - https://aws.amazon.com/apache2.0
 
 --------------------------------------------------------------------------------
 
-Group: software.amazon.awssdk  Name: sts  Version: 2.28.5
+Group: software.amazon.awssdk  Name: sts  Version: 2.30.6
 Project URL: https://aws.amazon.com/sdkforjava
 License: Apache License, Version 2.0 - https://aws.amazon.com/apache2.0
 
 --------------------------------------------------------------------------------
 
-Group: software.amazon.awssdk  Name: third-party-jackson-core  Version: 2.28.5
+Group: software.amazon.awssdk  Name: third-party-jackson-core  Version: 2.30.6
 Project URL: https://aws.amazon.com/sdkforjava
 License: Apache License, Version 2.0 - https://aws.amazon.com/apache2.0
 
 --------------------------------------------------------------------------------
 
-Group: software.amazon.awssdk  Name: utils  Version: 2.28.5
+Group: software.amazon.awssdk  Name: utils  Version: 2.30.6
 License: Apache License, Version 2.0 - https://aws.amazon.com/apache2.0
 
 --------------------------------------------------------------------------------
 
-Group: software.amazon.awssdk.crt  Name: aws-crt  Version: 0.30.6
+Group: software.amazon.awssdk.crt  Name: aws-crt  Version: 0.33.6
 Project URL: https://github.com/awslabs/aws-crt-java
 License: The Apache Software License, Version 2.0 - http://www.apache.org/licenses/LICENSE-2.0.txt
 
---------------------------------------------------------------------------------
-
-Group: software.amazon.eventstream  Name: eventstream  Version: 1.0.1
-Project URL: https://github.com/awslabs/aws-eventstream-java
```

Comment:
Can we double check before removing this, I still see these files. Are these classes coming from somewhere else? 
![Screenshot 2025-01-31 at 1 20 46 PM](https://github.com/user-attachments/assets/2e648ac7-a7a7-4538-81f9-9140b45013e0)

Reply:
Let me double check.

Reply:
I think we're bringing it transitively through the AWS v2 SDK https://github.com/aws/aws-sdk-java-v2/blob/2b988f9a2fc071d10a4eecd49d888b6b265e60da/core/http-auth-aws-eventstream/pom.xml#L43

but regardless if we bundle the classes, seems like we shouldn't remove this

Reply:
That's right. Let me re-add it.

Reply:
Fixed !